en/02_Developer_Guides/05_Extending/00_Modules.md

```diff
@@ -104,11 +104,11 @@ will constitute a project in itself. The below assumes you are using GitHub and
 Push your module upstream to the empty repository just created:
 
 ```bash
-	git init
-	git add -A
-	git commit -m 'first commit'
-	git remote add origin git@github.com:my_vendor/nice_feature.git
-	git push -u origin master
+git init
+git add -A
+git commit -m 'first commit'
+git remote add origin git@github.com:my_vendor/nice_feature.git
```

Comment:
```suggestion
git remote add origin git@github.com:my_vendor/module_name.git
```

Reply:
Things like this are out of scope or I'd be here forever. I'll make this change because you pointed it out but in general that's much more detail than I will be generally looking into.

Reply:
Fair enough, all good